resources/codegen_inputs/font.rs

```diff
@@ -29,3 +29,19 @@ record TableRecord {
     /// Length of the table.
     length: BigEndian<u32>,
 }
+
+/// [TTC Header](https://learn.microsoft.com/en-us/typography/opentype/spec/otff#ttc-header)
+table TTCHeader {
+    ttc_tag: BigEndian<Tag>,
+    #[version]
+    version: BigEndian<Version16Dot16>,
```

Comment:
This should be `MajorMinor`, not `Version16Dot16`.

Reply:
Thanks! I wasn’t aware of this type